pkg/controllers/managementuser/rbac/roletemplates/prtb_handler.go

```diff
@@ -77,8 +78,9 @@ func (p *prtbHandler) reconcileBindings(prtb *v3.ProjectRoleTemplateBinding) err
 	ownerLabel := createPRTBOwnerLabel(prtb.Name)
 
 	// Select all namespaces in project
+	_, projectId, _ := strings.Cut(prtb.ProjectName, ":")
```

Comment:
Is there any chance that `prtb.ProjectName` is malformed and this could be considered an error (this field is part of the user's input)? It's probably worth extracting this to a small function, which could return an error in such case.